src/commands/aem/rde/install.js

```diff
@@ -332,7 +334,7 @@ function guessType(name, url, pathFlag) {
     case '.zip':
       if (url.protocol === 'file:') {
         const zip = new Zip(fs.realpathSync(url), {});
-        const isContentPackage = zip.getEntry(JCR_ROOT + '/') !== null;
+        const isContentPackage = zip.getEntry(JCR_ROOT_SUBPATH) !== null;
```

Comment:
This is broken now. Changed from `jcr_root/` to `/jcr_root/`.

Leave this as-is, and for the other places, please use a utility like the one below.

```
function getPosAfterFolder(path, folder) {
  let pos = path.indexOf(folder + '/')
  if (pos == 0 || pos > 0 && path.charAt(pos - 1) === '/') {
    return pos + folder.length() + 1;
  }
  return -1;
}
```